lib/Controller/Schedule.php

```diff
@@ -2057,6 +2105,17 @@ public function edit(Request $request, Response $response, $id)
             }
         }
 
+        // Old layout cleanup
```

Comment:
Does it make sense to do this inside the transaction, or would it be better as a background task?